fe/topology.py

```diff
@@ -0,0 +1,305 @@
+import numpy as np
+import jax.numpy as jnp
+
+from timemachine.lib import potentials
+from ff.handlers import nonbonded, bonded
+
+_SCALE_12 = 1.0
+_SCALE_13 = 1.0
+_SCALE_14 = 0.5
+_BETA = 2.0
+_CUTOFF = 1.2
+
+class HostGuestTopology():
+
+    def __init__(self, guest_topology, host_p, num_host_atoms):
+        """
+        Utility tool for combining host with a guest, in that order.
+
+        Parameters
+        ----------
+        guest_topology:
+            Guest's Topology {Base, Dual, Single}Topology.
+
+        host_p:
+            Nonbonded potential for the host.
+
+        num_host_atoms:
+            Number of atoms in the host.
+    
+        """
+        self.guest_topology = guest_topology
+        self.host_p = host_p
+        self.num_host_atoms = num_host_atoms
+
+    def parameterize_harmonic_bond(self, ff_params):
```

Comment:
I agree this is a little verbose, but I want to avoid indirection as much as possible for these simple one-liners. We can refactor later if it becomes cumbersome.

Reply:
Sounds good!